style.css

```diff
@@ -330,7 +331,104 @@ li {
   color: #f55800;
 }
 
-.button:disabled {
-  background-color: #f4f1f0;
-  color: #5b5957;
+.form {
+  height: 844px;
+  background-color: #fff;
+  display: flex;
+  min-width: 375px;
+  flex-direction: column;
+  gap: 58px;
+  padding: 120px 24px 120px 24px;
+  justify-content: center;
+}
+
+/* .text-container {
+  min-width: 327px;
+  height: 216px;
+  display: flex;
+  align-items: center;
+} */
+
+.form-text {
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
+  color: #172b4d;
+  line-height: 44px;
+  text-align: center;
+}
+
+.form-form {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  align-items: center;
+  height: 330px;
+  margin-top: 0;
+}
```

Comment:
Kindly update the styling of your contact section to match the figma template better🙏

Your design:
<img width="156" alt="image" src="https://user-images.githubusercontent.com/37116322/221979530-87ceb7ff-2d12-4480-a616-c658364f2094.png">


Figma design:
<img width="180" alt="image" src="https://user-images.githubusercontent.com/37116322/221979666-f759a345-d603-488f-862e-d638ee9fff17.png">

A few things to note:
- Please reduce the size of the social links at the bottom and centralize them
- An element in your webpage is causing a horizontal scroll throughout your portfolio. I suggest going through the elements in the other sections and making sure each element doesn't overflow horizontally. Try checking the social links in the headline section
- Please make sure the get in touch button has interaction styling when the button is hovered and pressed. You can do this using the `:hover` and `:active` pseudo-selectors. Please make sure to follow the style guide by figma
<img width="355" alt="image" src="https://user-images.githubusercontent.com/37116322/221981241-45b12e8d-b80f-41f9-9c63-ebdea087a932.png">